src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -632,12 +657,290 @@ void copy_columns_by_row_chunks(const ov::Tensor& src, ov::Tensor& dst) {
     }
 }
 
+enum NpuPipeline {
```

Comment:
How about these:

- `NPUPipelineType`
- `NPUPipelineKind`
- `NPUPipelineImpl`

Reply:
WHY NPU AGAIN

Reply:
Anyways, we do pass NPUW options directly in the pipeline code, so why not. Everything works but not `Npu` please.

Reply:
`StaticPipelineType`, etc

Reply:
Should the option `NPU_PIPELINE` be changed also to `STATIC_PIPELINE`?